sherpa-onnx/c-api/c-api.h

```diff
@@ -101,8 +101,33 @@ SHERPA_ONNX_API typedef struct SherpaOnnxOnlineRecognizerConfig {
 } SherpaOnnxOnlineRecognizerConfig;
 
 SHERPA_ONNX_API typedef struct SherpaOnnxOnlineRecognizerResult {
+  // Recognized text
   const char *text;
-  // TODO(fangjun): Add more fields
+
+  // Pointer to continuous memory which holds string based tokens
+  // which are seperated by \0
+  const char *tokens;
```

Comment:
I suggest that we define
```
const char *_tokens;
const char *const*tokens;
```

where `_tokens` is your current `tokens`

And the new `char **tokens` is a pointer array.
`tokens[0]` contains the address of the first `token` in
`_tokens`. `tokens[1]` contains the address of the second
`token` in `_tokens`.

In this way, it simplifies users' life as they only need to iterate
`char **tokens`.

```
for (int32 i = 0; i != count; ++i) {
  const char*t = tokens[i];
  // process this token
}
```

Reply:
@csukuangfj  I added `const char *const *tokensArr;` for easy to implement, and I also tested